My Project/clsWinformsComponents.vb

```diff
@@ -0,0 +1,183 @@
+﻿' IDEMS International
+' Copyright (C) 2021
+'
+' This program is free software: you can redistribute it and/or modify
+' it under the terms of the GNU General Public License as published by
+' the Free Software Foundation, either version 3 of the License, or
+' (at your option) any later version.
+'
+' This program is distributed in the hope that it will be useful,
+' but WITHOUT ANY WARRANTY; without even the implied warranty of
+' MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License 
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Imports System.ComponentModel
+Imports System.Windows.Forms
+Imports System.Text.RegularExpressions
+
+
+'''------------------------------------------------------------------------------------------------
+''' <summary>
+''' Provides utility functions for getting winforms components
+''' <para>
+''' Note: This class is intended to be used solely as a 'static' class (i.e. contains only shared 
+''' members, cannot be instantiated and cannot be inherited from).
+''' In order to enforce this (and prevent developers from using this class in an unintended way), 
+''' the class is declared as 'NotInheritable` and the constructor is declared as 'Private'.</para>
+''' </summary>
+'''------------------------------------------------------------------------------------------------
+Public NotInheritable Class clsWinformsComponents
```

Comment:
make class name consistent with other classes in package?
```suggestion
Public NotInheritable Class clsWinFormComponents
```
Should we also change the file name?

This comment is minor, feel free to ignore it if you don't think the change is worth it